.github/workflows/e2e.yaml

```diff
@@ -86,6 +89,4 @@ jobs:
         uses: ./
         with:
           tags: ${{ steps.metadata.outputs.tags }}
-          suffixes: |
-            -linux-amd64
-            -linux-arm64
+          digests: ${{ join(needs.*.outputs.digest) }}
```

Comment:
This does not contain all jobs.
https://docs.github.com/en/actions/learn-github-actions/contexts#needs-context